### DISCUSSION TITLE
opa-bundle-error

### DISCUSSION THREAD

**Discussion by rohithk2301** [open-policy-agent/community#308] [OPA and Rego] [answered]
HI , 
Iam getting below error:

"/usr/share/nginx/html/bundle.tar.gz" failed (2: No such file or directory), client: 172.18.0.3, server: localhost, request: "GET /bundle.tar.gz HTTP/1.1", host: "bundle_server"
opa_1            | {
opa_1            |   "level": "error",
opa_1            |   "msg": "Bundle load failed: server replied with Not Found",
opa_1            |   "name": "nginx",
opa_1            |   "plugin": "bundle"

and My docker compose file looks like as below:

services:
  opa:
    image: openpolicyagent/opa:0.40.0-rootless
    ports:
      - "8181:8181"
    command:
      - "run"
      - "--server"
      - "--log-format=json-pretty"
      - "--set=decision_logs.console=true"
      - "--set=services.nginx.url=http://bundle_server"
      - "--set=bundles.nginx.service=nginx"
      - "--set=bundles.nginx.resource=bundle.tar.gz"
      - "--set=bundles.nginx.polling.min_delay_seconds=10"
      - "--set=bundles.nginx.polling.max_delay_seconds=30"
    depends_on:
      - bundle_server
  api_server:
    image: openpolicyagent/demo-restful-api:0.2
    ports:
      - "5000:5000"
    environment:
      - OPA_ADDR=http://opa:8181
      - POLICY_PATH=/v1/data/httpapi/authz
    depends_on:
      - opa
  bundle_server:
    image: nginx:1.20.0-alpine
    ports:
      - 8888:80
    volumes:
      - ../:/usr/share/nginx/html/

**Comment by anderseknert** [comment, marked as answer]
""/usr/share/nginx/html/bundle.tar.gz" failed (2: No such file or directory)"  makes it seem like there is no bundle present in the directory, or that the process is missing permissions to read it.
